app/js/components/labOrderEntry/LabEntryForm.jsx

```diff
@@ -14,46 +18,54 @@ export class LabEntryForm extends React.Component {
     categoryId: 1,
     defaultTests: [],
     selectedTests: [],
-    selectedPanel: null,
+    selectedPanelIds: [],
     disableSaveButton: true,
```

Comment:
there is no need to use default values when you got the default value of `draftLabOrders` in redux already, so just use the length and get the value

Reply:
It has been removed.